versioned/tiered/dynamodb/src/main/java/org/projectnessie/versioned/dynamodb/DynamoStore.java

```diff
@@ -570,4 +598,14 @@ private static void verifyKeySchema(TableDescription description) {
     throw new IllegalStateException(String.format("Invalid key schema for table: %s. Key schema should be a hash partitioned "
         + "attribute with the name 'id'.", description.tableName()));
   }
+
+  private Tracer getTracer() {
```

Comment:
we can only use the global tracer here but above we have the option to specify a tracer. Can we do one or the other? I prefer just using the global tracer.

Reply:
It's been due to `GlobalTracer` not allowíng us to register the "global tracer" once, but never change it. I've updated the tests to use a proxy-instance between the `Tracer` for the tests and `GlobalTracer`